tests/scripts/minikube.sh

```diff
@@ -51,11 +51,11 @@ copy_image_to_cluster() {
 }
 
 # configure dind-cluster
-KUBE_VERSION=${KUBE_VERSION:-"v1.7.2"}
+KUBE_VERSION=${KUBE_VERSION:-"v1.7.0"}
```

Comment:
actually, it looks like today that up to 1.7.3 is now supported by minikube!
```
curl -s https://storage.googleapis.com/minikube/k8s_releases.json
[
  {
    "version": "v1.7.3"
  },
  {
    "version": "v1.7.2"
  },
  {
    "version": "v1.7.0"
  },
  {
    "version": "v1.7.0-rc.1"
  },
  {
    "version": "v1.7.0-alpha.2"
  },
<omitted older versions>
```
so we can use whatever up to 1.7.3 now, which is probably good because that supports CRDs.